README.md

```diff
@@ -2,9 +2,9 @@
 
 [![CI](https://github.com/silverstripe/doorman/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/doorman/actions/workflows/ci.yml)
 
-Child process management. Compatible from PHP `5.3` to PHP `7`. Needs no extensions.
+Child process management. Needs no extensions.
 
-> `2.x` supports PHP `5.5.9` and upwards. If you need PHP `5.3/4` support, use a `1.x` release.
+The is currently only intended for use with Silverstripe CMS.
```

Comment:
```suggestion
This currently only intended for use with Silverstripe CMS.
```
Small typo - but also I don't know what the purpose of that note is. Was this a fork of something and we had to adopt it? Or does it have any silverstripe cms specific code?

Reply:
Removed